index.html

```diff
@@ -10,62 +10,62 @@
 
 </head>
 <body>
+    <!-- added an section aka s Container.. to contain the grid -->
+    <section class="sContainer">
     <div class="crew_title">
         <h1>The Straw Hat Pirates</h1>
     </div>
-
+    <!-- add onClick() here on crew_button to display
+    crew description -->
     <div class="crew_container">
-        <h2 class="crew_subtitle">Who We Are</h2>
+        <button class="crew_button">Who We Are</button>
         <p class="crew_description">The Straw Hats are pirates who don't abide by the laws of the World Government. We are a very infamous and powerful pirate crew that originated from the East Blue looking for adventure and the One Piece. Although we're considered villains by the World Government and the general public who are not familiar with us, we are kindhearted for the most part and only fight in self-defense or vengeance for our friends. Our current bounties now total up to 8,816,001,000 Berries.
         </p>
     </div>
 
     <!-- ALL Crew Members-->
     <div class="crew_all" class="img">
-
          <!-- Crew Members Left-->
         <div class="crew_members_left">
             <div class="crew_captain">
                 <img class="luffy_img" src="images/luffy.jpeg">
                 <p class="names">Monkey D. Luffy</p>
-                <p>Captain</p>
+                <p class="titles">Captain</p>
```

Comment:
I like that you added in the class here for all of them. Will be much easier to style!